src/libs/SelectionScraper/index.ts

```diff
@@ -138,16 +143,15 @@ const replaceNodes = (dom, isChildOfEditorElement) => {
         name: domName,
         attribs: domAttribs,
         children: domChildren,
-    };
+    } as unknown as Node;
```

Comment:
@fabioh8010 This assertion was my idea, other thing that we could do is to return `new Element(...)` or `new DataNode(...)` instead of assertion, but I didn't want to change the logic that much, wdyt?

Reply:
```suggestion
    } as Element & DataNode;
```

How about this? :)

Reply:
Definitely better